app/forms/ConsignmentReferences.scala

```diff
@@ -16,59 +16,58 @@
 
 package forms
 
+import forms.ConsignmentReferenceType.ConsignmentReferenceType
 import forms.EnhancedMapping.requiredRadio
 import play.api.data.Forms.text
 import play.api.data.{Form, Forms}
-import play.api.libs.json.Json
+import play.api.libs.json.{Format, Json, Reads, Writes}
 import uk.gov.voa.play.form.ConditionalMappings.mandatoryIfEqual
 import utils.FieldValidator._
 
-case class ConsignmentReferences(reference: String, referenceValue: String)
+case class ConsignmentReferences(reference: ConsignmentReferenceType, referenceValue: String)
+
+object ConsignmentReferenceType extends Enumeration {
+  type ConsignmentReferenceType = Value
+  val D, M = Value
```

Comment:
Lets stick with Ducr and Mucr with value - makes code less challenging to read

Reply:
Specially if those would be imported into scope

Reply:
It'll involve a bunch of custom Reads & Writes for only minimal gain. Perhaps it can be one for the tech debt board. It's still better than what we have today :)

Reply:
I'll add a formatter test though so it's easier to do